web/src/App.vue

```diff
@@ -1,21 +1,41 @@
 <template lang="pug">
 v-app
-  v-toolbar.main-toolbar.darken-4(dark, dense, color="primary")
-    v-toolbar-items
-      v-btn.px-0.square-button.darken-4(flat, color="white", to="/select")
-        v-icon {{ $vuetify.icons.close }}
-    v-spacer
-    v-toolbar-title.title
-      span.font-weight-bold.secondary--text.text--lighten-3 Biomarker Analysis »
-      span.font-weight-bold.white--text  {{ $router.currentRoute.name }}
-    v-spacer
+  v-toolbar.darken-3(dense, dark, color="primary")
+    v-toolbar-side-icon.logo(@click="$router.push('/')") V
+    v-breadcrumbs(:items="breadcrumbs", divider="»")
+      template(#item="props")
+        v-breadcrumbs-item(:to="props.item.to", exact) {{props.item.text}}
   v-container.pa-0.d-flex(fluid)
     router-view.grow
 </template>
 
 <script>
+
 export default {
   name: 'App',
+  computed: {
+    breadcrumbs() {
+      return [
+        {
+          text: 'VIIME',
+          to: '/',
```

Comment:
Going forward, please prefer [named routes](https://router.vuejs.org/guide/essentials/named-routes.html) instead of absolute ones.  I'll have to go back and fix them as mentioned in #251 but